starter_files/lib/blackjack_game.rb

```diff
@@ -0,0 +1,140 @@
+require_relative 'deck'
+require_relative 'player'
+require_relative 'hand'
+require_relative 'card'
+require_relative 'dealer'
+
+class BlackjackGame
+
+  attr_reader :player
+
+  def initialize
+    @player = Player.new(100)
+    @dealer = Dealer.new
```

Comment:
I like how you are initializing the player with their starting allotment of money. Good idea!